### DISCUSSION TITLE
ZimGPT Plugin

### DISCUSSION THREAD

**Discussion by datenwurm** [zim-desktop-wiki/zim-desktop-wiki#2360] [Show and tell]
Hello Zim-Community,

I'm excited to share my new ZimGPT plugin with you, which integrates ChatGPT into Zim and allows you to generate text directly in your Zim notes.

I'm aware that there was a previous discussion about a similar plugin, and I want to clarify that my ZimGPT plugin is a separate and independent project. I developed it from scratch to address a specific need, and I hope it can be useful to others as well.

Using ZimGPT is easy: simply copy the plugin into your plugin directory (detailed instructions can be found in the README.md file), activate the plugin, and enter your API key in the plugin preferences.

Once set up, just select some text and press alt+shift+i to open the dialog where you can enter your instruction or question for ChatGPT to generate text based on your selection.

I'd love to hear your feedback on ZimGPT and how it could be improved. You can download the plugin from my GitHub repository at https://github.com/datenwurm/zim-plugin-zim-gpt.

**Comment by introt** [comment]
Link to the aforementioned previous discussion: #2350
